config/core.php

```diff
@@ -47,7 +47,7 @@
         [
             'type'    => 'plugin',
             'label'   => 'OpenPDC Base',
-            'version' => '2.1.5',
+            'version' => '3.13.6',
```

Comment:
Vanaf [deze](https://github.com/OpenWebconcept/plugin-pdc-base/commits/v3.13.6/) versie werkte de translations weer in de PDC-base en aangezien de Plugin.php van deze plug-in die van de PDC-base extend is deze versie vereist.